src/advanced/domains/cart/CartDisplay.tsx

```diff
@@ -0,0 +1,159 @@
+import { useCallback } from "react";
+import { ICartItem } from "./types";
```

Comment:
I 가 인터페이스라서 요렇게 작성해주신 거죵? 저는 이런 것도 제안드려봅니다.

```suggestion
import { CartItem as CartItemType } from "./types";
```